src/common/win32/EmuShared.h

```diff
@@ -286,6 +291,16 @@ class EmuShared : public Mutex
 		void GetOverlaySettings(overlay_settings *value) { Lock(); *value = m_imgui_overlay_settings; Unlock(); }
 		void SetOverlaySettings(const overlay_settings* value) { Lock(); m_imgui_overlay_settings = *value; Unlock(); }
 
+		// ******************************************************************
+		// * Git version Accessor (only the get method is provided because it should not be changed
```

Comment:
```suggestion
		// * Git version Accessor (only the get method is provided because it should not be changed)
```
Missing closing bracket.